ci-operator/step-registry/ipi/aws/ipi-aws-workflow.yaml

```diff
@@ -4,6 +4,7 @@ workflow:
     allow_best_effort_post_steps: true
     pre:
     - chain: ipi-aws-pre
+    - ref: cucushift-installer-reportportal-marker
```

Comment:
This install workflow is shared by many non-qe ci jobs, I am not sure if it is a good idea to enable ReportPortal steps for it in case it bring some other noises.